### ISSUE TITLE
Problem with `.` in module name?

### ISSUE THREAD

**Issue by jjeff** [RSeidelsohn/license-checker-rseidelsohn#5]
First off, thanks for updating license-checker! I had found this bug in license-checker as well, but when I went to report it, I quickly found that it was unmaintained. So I was happy when I found your fork.

I'm using the [wavesurfer.js package](https://www.npmjs.com/package/wavesurfer.js?activeTab=readme) in my project and it doesn't seem to be showing up in the license-checker-rseidelsohn output. I've never seen another npm package with `.` in its name, so I wonder if that might be why it's not showing up.

**Comment by RSeidelsohn** [comment]
Ok, thanks for filing this issue. I'll have a look, and I am as well astonished as happy about the fact that other people use my fork!

**Comment by RSeidelsohn** [comment]
Hi @jjeff , I just tried to verify your issue, but can't reproduce it. The reason why might be the fact that I just finished upgrading old dependencies from `license-checker-rseidelsohn` (sorry for the ugly name - I will probably take care of this in the future).
I created a new project with only the `wavesurfer.js` and the `license-checker-rseidelsohn` dependency and ran the license-checker. The output was:
![Peek 2021-04-07 14-42](https://user-images.githubusercontent.com/870515/113868105-992f0a80-97af-11eb-9f9b-2bec78a73edf.gif)

So I kindly ask you to try the latest version and tell me if it now is fine or if you still have this issue.

Cheers, Roman.
